chirp/drivers/kguv9dplus.py

```diff
@@ -1808,20 +2161,25 @@ def _key_tab(self):
         kf = RadioSettingGroup("key_grp", "Key Settings")
 
         kf.append(RadioSetting("settings.pf1",
-                               "PF1 Key function(55)",
+                               "PF1 Key function (Menu 55)",
                                RadioSettingValueList(
                                    PF1KEY_LIST,
                                    PF1KEY_LIST[s.pf1])))
         kf.append(RadioSetting("settings.pf2",
-                               "PF2 Key function(56)",
+                               "PF2 Key function (Menu 56)",
                                RadioSettingValueList(
                                    PF2KEY_LIST,
                                    PF2KEY_LIST[s.pf2])))
+        if self.MODEL == "KG-UV9PX":
+            pfkey3 = PF3KEY_LIST2
```

Comment:
I think it would be better to rename this to `PF3KEY_UV9PX` or something. If someone comes later the change the names up above where it's defined, they might accidentally edit the one they don't intend because `LIST` and `LIST2` are not very different.

Reply:
agree... will update it in a future clean-up